API/projectClass/projectClass.m

```diff
@@ -152,36 +152,14 @@ function delete(obj)
             domainsObj = obj.domainsClass();
         end
 
-        function obj = setUsePriors(obj, showFlag)
-            % Sets the use priors flag. The showFlag should be a boolean/logical.
-            %
-            % project.setUsePriors(true); 
-            if ~islogical(showFlag)
-                throw(exceptions.invalidType('usePriors must be logical ''true'' or ''false'''));
+        function set.showPriors(obj, flag)
+            % Setter for the showPriors property. It indicate priors 
+            % should be visible when printing the project. The flag should 
+            % be a boolean/logical.
+            if ~islogical(flag)
+                throw(exceptions.invalidType('Show priors must be true or false'));
```

Comment:
I don't think you can use a argument block in a setter

Since it is used as a variable `problem.showPriors = True` I don't know how you will call the setter without a value

Reply:
I tried this it did not throw any exception when I set flag to a string so I will keep as it is for now 
```
        function set.showPriors(obj, flag)
            arguments
                obj
                flag {logical}
            end
            obj.showPriors = flag;
        end
```